crates/core/src/host/module_host.rs

```diff
@@ -145,151 +144,126 @@ impl From<DatabaseTableUpdate> for TableUpdateJson {
 }
 
 #[derive(Debug)]
-pub struct DatabaseUpdateCow<'a> {
-    pub tables: Vec<DatabaseTableUpdateCow<'a>>,
+pub struct DatabaseUpdateRV<'a> {
```

Comment:
I'm less mad about the abbreviation here, but I would prefer if we just wrote `RelValue`. Failing that, I think we should follow Rust convention for camel-casing acronyms, i.e. do `DatabaseUpdateRv`.